docs/reference/css_guidelines.rst

```diff
@@ -0,0 +1,151 @@
+Contributing CSS to Wagtail
+===========================
+
+Our CSS is written in Sass, using the SCSS syntax.
+
+Spacing
+~~~~~~~
+
+-  Use soft-tabs with a four space indent. Spaces are the only way to
+   guarantee code renders the same in any person's environment.
+-  Put spaces after ``:`` in property declarations.
+-  Put spaces before ``{`` in rule declarations.
+-  Put line breaks between rulesets.
+-  When grouping selectors, keep individual selectors to a single line.
+-  Place closing braces of declaration blocks on a new line.
+-  Each declaration should appear on its own line for more accurate
+   error reporting.
+-  Add a newline at the end of your ``.scss`` files.
+-  Strip trailing whitespace from your rules.
+
+Formatting
+~~~~~~~~~~
+
+-  Use hex color codes ``#000`` unless using ``rgba()`` in raw CSS
+   (SCSS' ``rgba()`` function is overloaded to accept hex colors as a
+   param, e.g., ``rgba(#000, .5)``).
+-  Use ``//`` for comment blocks (instead of ``/* */``).
+-  Use single quotes for string values
+   ``background: url('my/image.png')`` or ``content: 'moose'``
+-  Avoid specifying units for zero values, e.g., ``margin: 0;`` instead
+   of ``margin: 0px;``.
+-  Strive to limit use of shorthand declarations to instances where you
+   must explicitly set all the available values.
+
+Sass imports
+~~~~~~~~~~~~
+
+Leave off underscores and file extensions in includes:
+
+.. code-block:: scss
+
+    // Bad
+    @import 'components/_widget.scss'
+
+    // Better
+    @import 'components/widget'
+
+Pixels vs. ems
+~~~~~~~~~~~~~~
+
+Use ``rems`` for ``font-size`` with a ``px`` fallback, because it offers
```

Comment:
Is a px fallback necessary? http://caniuse.com/#search=rems Wagtail only supports IE9+ and despite the `font` shorthand not working, as far as I'm aware there's no need for fallbacks. It would certainly be preferable to not clutter the code with them if possible,